frontend/src/features/properties/map/research/add/models.ts

```diff
@@ -35,76 +34,9 @@ export class ResearchForm {
     const newForm = new ResearchForm();
     newForm.id = model.id;
     newForm.name = model.fileName || '';
-    newForm.properties = model.researchProperties?.map(x => PropertyForm.fromApi(x)) || [];
-    newForm.rowVersion = model.rowVersion;
-
-    return newForm;
-  }
-}
-
-export class PropertyForm {
```

Comment:
This model was not doing anything specific to research files, so moved this into the common area and reused it between acquisition and research files instead of using the duplicated model.